drivers/xen/events.c

```diff
@@ -47,6 +47,22 @@ int alloc_unbound_event_channel(domid_t remote_dom)
 	return rc;
 }
 
+int alloc_unbound_event_channel_dom0(domid_t dom, domid_t remote_dom)
```

Comment:
Please correct me if I am wrong, but can this be protected with the dom0 Kconfig? Would it make sense?